app/modules/transcribe/transcribe.controller.js

```diff
@@ -78,7 +78,8 @@ function TranscribeController($stateParams, $modal, $scope, $window, Annotations
     function loadSubject() {
         return SubjectsFactory.$getData($stateParams.subjectSet)
             .then(subjectLoaded, subjectLoadError)
```

Comment:
`subjectLoadError` here will catch any errors, then run the next `then` block. I don't think this is what you intend this piece of code to do?